seqr/migrations/0064_alter_phenotypeprioritization.py

```diff
@@ -0,0 +1,56 @@
+# Generated by Django 3.2.25 on 2024-05-02 17:45
+from django.conf import settings
+from django.db import migrations, models
+import django.utils.timezone
+
+from seqr.models import _slugify
+
+MAX_GUID_SIZE = 30
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('seqr', '0063_dynamicanalysisgroup'),
```

Comment:
This is what I was referring to when I said you might have to update the migration, but since it looks like it was already depending on the last migration I added it makes sense that there was no further change needed!